libs/components/manifest/project.json

```diff
@@ -5,7 +5,7 @@
   "projectType": "library",
   "tags": ["npm"],
   "targets": {
-    "build": {
+    "prebuild": {
```

Comment:
Flipping the build/postbuild scripts for the manifest to prebuild/build so that running `nx build manifest` both builds the manifest library, but also generates the manifest JSON files. This is useful for local development, such as the code examples playground, and the custom ESLint rules which require the JSON files exist before they can be properly consumed.